packages/stark-testing/karma.conf.js

```diff
@@ -38,7 +38,8 @@ const rawKarmaConfig = {
 			],
 			resolve: {},
 			transforms: [
-				require("karma-typescript-angular2-transform") // see https://github.com/monounity/karma-typescript-angular2-transform
+				require("karma-typescript-angular2-transform"), // see https://github.com/monounity/karma-typescript-angular2-transform
+				require("karma-typescript-es6-transform")() // add preset in a way that subpackages can find it
```

Comment:
Can you also add the `babel-preset` here the same way you added it in stark-core and stark-ui? `presets: [helpers.root("../stark-testing/node_modules/babel-preset-env")]`

 I know it is taken by default from `node_modules/babel-preset-env` but it is easier to understand what needs to be modified by looking at the options passed to the `"karma-typescript-es6-transform`. And that will be really useful for developers if they check this `karma.conf` file in stark-testing